[Timing Build Error] timeFrobeniusFactor.cpp: no matching function call

<!--Please only submit issues/bug reports that come with enough information to reproduce them, ideally a unit test that fails, and possible ideas on what might be wrong. -->

<!-- Even better yet, fix the bug and/or documentation, add a unit test, and create a pull request! -->

<!-- This is a channel to report bugs/issues, not a support channel to help install/use/debug your own code. We'd love to help, but just don't have the bandwidth. Please post questions in the GTSAM Google group (https://groups.google.com/forum/#!forum/gtsam-users) -->

## Description

When attempting to compile the `timing` scripts, `timeFrobeniusFactor` fails with the following error:

```
[ 81%] Building CXX object timing/CMakeFiles/timeFrobeniusFactor.dir/timeFrobeniusFactor.cpp.o
In file included from /usr/include/c++/10.1.0/x86_64-pc-linux-gnu/bits/c++allocator.h:33,
                 from /usr/include/c++/10.1.0/bits/allocator.h:46,
                 from /usr/include/c++/10.1.0/memory:64,
                 from /usr/include/tbb/tbb_stddef.h:452,
                 from /usr/include/tbb/tbb_allocator.h:20,
                 from /home/acxz/vcs/git/github/borglab/gtsam/gtsam/base/FastDefaultAllocator.h:34,
                 from /home/acxz/vcs/git/github/borglab/gtsam/gtsam/base/FastMap.h:21,
                 from /home/acxz/vcs/git/github/borglab/gtsam/gtsam/base/timing.h:20,
                 from /home/acxz/vcs/git/github/borglab/gtsam/timing/timeFrobeniusFactor.cpp:19:
/usr/include/c++/10.1.0/ext/new_allocator.h: In instantiation of ‘void __gnu_cxx::new_allocator<_Tp>::construct(_Up*, _Args&& ...) [with _Up = gtsam::FrobeniusWormholeFactor; _Args = {const long unsigned int&, const long unsigned int&, gtsam::SO<3>, const boost::shared_ptr<gtsam::noiseModel::Base>&}; _Tp = gtsam::FrobeniusWormholeFactor]’:
/usr/include/c++/10.1.0/bits/alloc_traits.h:247:17:   required from ‘static constexpr std::_Require<typename std::allocator_traits<_Alloc>::__construct_helper<_Tp, _Args>::type> std::allocator_traits<_Alloc>::_S_construct(_Alloc&, _Tp*, _Args&& ...) [with _Tp = gtsam::FrobeniusWormholeFactor; _Args = {const long unsigned int&, const long unsigned int&, gtsam::SO<3>, const boost::shared_ptr<gtsam::noiseModel::Base>&}; _Alloc = Eigen::aligned_allocator<gtsam::FrobeniusWormholeFactor>; std::_Require<typename std::allocator_traits<_Alloc>::__construct_helper<_Tp, _Args>::type> = void; typename std::allocator_traits<_Alloc>::__construct_helper<_Tp, _Args>::type = std::integral_constant<bool, true>]’
/usr/include/c++/10.1.0/bits/alloc_traits.h:360:16:   required from ‘static decltype (std::allocator_traits<_Alloc>::_S_construct(__a, __p, (forward<_Args>)(std::allocator_traits<_Alloc>::construct::__args)...)) std::allocator_traits<_Alloc>::construct(_Alloc&, _Tp*, _Args&& ...) [with _Tp = gtsam::FrobeniusWormholeFactor; _Args = {const long unsigned int&, const long unsigned int&, gtsam::SO<3>, const boost::shared_ptr<gtsam::noiseModel::Base>&}; _Alloc = Eigen::aligned_allocator<gtsam::FrobeniusWormholeFactor>; decltype (std::allocator_traits<_Alloc>::_S_construct(__a, __p, (forward<_Args>)(std::allocator_traits<_Alloc>::construct::__args)...)) = void]’
/usr/include/boost/smart_ptr/make_shared_object.hpp:289:41:   required from ‘typename boost::detail::sp_if_not_array<T>::type boost::allocate_shared(const A&, Args&& ...) [with T = gtsam::FrobeniusWormholeFactor; A = Eigen::aligned_allocator<gtsam::FrobeniusWormholeFactor>; Args = {const long unsigned int&, const long unsigned int&, gtsam::SO<3>, const boost::shared_ptr<gtsam::noiseModel::Base>&}; typename boost::detail::sp_if_not_array<T>::type = boost::shared_ptr<gtsam::FrobeniusWormholeFactor>]’
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/inference/FactorGraph.h:169:61:   required from ‘gtsam::FactorGraph<FACTOR>::IsDerived<DERIVEDFACTOR> gtsam::FactorGraph<FACTOR>::emplace_shared(Args&& ...) [with DERIVEDFACTOR = gtsam::FrobeniusWormholeFactor; Args = {const long unsigned int&, const long unsigned int&, gtsam::SO<3>, const boost::shared_ptr<gtsam::noiseModel::Base>&}; FACTOR = gtsam::NonlinearFactor; gtsam::FactorGraph<FACTOR>::IsDerived<DERIVEDFACTOR> = void]’
/home/acxz/vcs/git/github/borglab/gtsam/timing/timeFrobeniusFactor.cpp:77:62:   required from here
/usr/include/c++/10.1.0/ext/new_allocator.h:150:4: error: no matching function for call to ‘gtsam::FrobeniusWormholeFactor::FrobeniusWormholeFactor(const long unsigned int&, const long unsigned int&, gtsam::SO<3>, const boost::shared_ptr<gtsam::noiseModel::Base>&)’
  150 |  { ::new((void *)__p) _Up(std::forward<_Args>(__args)...); }
      |    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/acxz/vcs/git/github/borglab/gtsam/timing/timeFrobeniusFactor.cpp:31:
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/slam/FrobeniusFactor.h:135:3: note: candidate: ‘gtsam::FrobeniusWormholeFactor::FrobeniusWormholeFactor(gtsam::Key, gtsam::Key, const gtsam::Rot3&, size_t, const SharedNoiseModel&)’
  135 |   FrobeniusWormholeFactor(Key j1, Key j2, const Rot3& R12, size_t p = 4,
      |   ^~~~~~~~~~~~~~~~~~~~~~~
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/slam/FrobeniusFactor.h:135:55: note:   no known conversion for argument 3 from ‘gtsam::SO<3>’ to ‘const gtsam::Rot3&’
  135 |   FrobeniusWormholeFactor(Key j1, Key j2, const Rot3& R12, size_t p = 4,
      |                                           ~~~~~~~~~~~~^~~
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/slam/FrobeniusFactor.h:128:20: note: candidate: ‘gtsam::FrobeniusWormholeFactor::FrobeniusWormholeFactor(const gtsam::FrobeniusWormholeFactor&)’
  128 | class GTSAM_EXPORT FrobeniusWormholeFactor : public NoiseModelFactor2<SOn, SOn> {
      |                    ^~~~~~~~~~~~~~~~~~~~~~~
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/slam/FrobeniusFactor.h:128:20: note:   candidate expects 1 argument, 4 provided
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/slam/FrobeniusFactor.h:128:20: note: candidate: ‘gtsam::FrobeniusWormholeFactor::FrobeniusWormholeFactor(gtsam::FrobeniusWormholeFactor&&)’
/home/acxz/vcs/git/github/borglab/gtsam/gtsam/slam/FrobeniusFactor.h:128:20: note:   candidate expects 1 argument, 4 provided
make[3]: *** [timing/CMakeFiles/timeFrobeniusFactor.dir/build.make:82: timing/CMakeFiles/timeFrobeniusFactor.dir/timeFrobeniusFactor.cpp.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:22661: timing/CMakeFiles/timeFrobeniusFactor.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2830: CMakeFiles/timing.dir/rule] Error 2
make: *** [Makefile:242: timing] Error 2
```

## Steps to reproduce

1. git clone
2. mkdir build && cd build && cmake .. && make timing

<!-- If you have a code sample, unit test, error messages, stack traces, etc., please provide it here as well -->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

`timeFrobeniusFactor` should compile successfully.

## Environment

<!-- Please provide information about your code environment, things such as OS, language of use (C++, Matlab or Python), version of dependent libraries if using a custom build etc., anything to provide us more information. -->

<!-- The more the information, the faster we can help resolve the issue -->

OS: Arch Linux
gtsam: latest develop head
gcc: 10.1.0

## Additional information

<!-- Add any other infor or context about the problem here. -->

N/A

--- a/timing/timeFrobeniusFactor.cpp
+++ b/timing/timeFrobeniusFactor.cpp
@@ -74,7 +74,7 @@ int main(int argc, char* argv[]) {
     const auto& Tij = factor->measured();
     const auto& model = factor->noiseModel();
     graph.emplace_shared<FrobeniusWormholeFactor>(
-        keys[0], keys[1], SO3(Tij.rotation().matrix()), model);
+        keys[0], keys[1], Rot3(Tij.rotation().matrix()), 4, model);
   }
 
   std::mt19937 rng(42);

